doc/source/examples/permutation_importance_classification_leave.ipynb

```diff
@@ -0,0 +1,933 @@
+{
```

Comment:
<p>Need a comment for the reader somewhere in this notebook about what <code>sales</code> means as otherwise it's quite illegible - I suspect it's a feature naming issue. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/798/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>Not sure what comment to add. I rephrased it so I hope now is more intelligible.</p>